_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,250 @@
+---
+id: b4f0c3
+name: meta viewport does not prevent zoom
+rule_type: atomic
+description: |
+  This rule checks that the `meta` element is not used to block the user agent ability to zoom.
+accessibility_requirements:
+  wcag21:1.4.4: # Resize text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to each `meta` element with a `name` attribute whose value is equal to [`viewport`](https://www.w3.org/TR/css-device-adapt-1/#viewport-meta) and has a `content` attribute.
```

Comment:
"whose value is equal to `viewport`" => "whose value is a case-insensitive match for `viewport`".

Reply:
Good catch 👍 Also, "case-insensitive" -> "[ASCII case-insensitive](https://infra.spec.whatwg.org/#ascii-case-insensitive)".